app/src/main/java/com/medmapper/v33001/dto/User.kt

```diff
@@ -2,5 +2,5 @@ package com.medmapper.v33001.dto
 
 data class User(
     val uid: String = "",
-    var displayName: String) {
-}
\ No newline at end of file
+    var displayName: String? = null
+)
```

Comment:
Consider making the displayName property nullable to avoid initializing it with an empty string.

Source: https://kotlinlang.org/docs/null-safety.html

Reply:
Huh.   I could go either way on this.

We generally try to avoid nullable types.

But then again... is empty string adding any value over a nullable?